operator/cmd/mesh/root.go

```diff
@@ -79,6 +79,7 @@ func GetRootCmd(args []string) *cobra.Command {
 	rootCmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
 
 	rootCmd.AddCommand(ManifestCmd(log.DefaultOptions()))
+	rootCmd.AddCommand(InstallCmd(log.DefaultOptions()))
```

Comment:
why would this be required?

Reply:
The `install` is under `ManifestCmd` which requires to attache `manifest install` in order to call it from test. otherwise it throws following error.
```go
<*errors.errorString | 0xc0005e1cf0>: {
                s: "unknown command \"install\" for \"mesh\"",
            }
```
In order for `install` to work directly in the test, it needs to be connected to the `mesh` command.

Reply:
I am not sure we want to add `manifest install` for real users just for this test?

Reply:
`istioctl manifest install` still works and is kept for backward compatibility. This code does not add `manifest install` it just makes it accessible in the test.